### PR TITLE
disable new tab at background by default now

### LINKED ISSUE #656: [Demand] Scheduling Revamp
## Metadata

Field | Value
--- | ---
Previous Stage | N/A
Figma Link | [figma.com/...](___)
Expected engineering due | 2020-01-22 (WED)
Corresponding Pull Request | -
Completion Version | -

## Content

Here we shall design a new scheduling mechanism to avoid opening too many sessions simultaneously.

Screen Shot 2020-01-19 at 06 05 22

## Notes

- None.

### PR DESCRIPTION
**Pull request by Jack-Works**
close #656

### CODE REVIEW

File: src/_locales/zh/messages.json [hunk 1]
```diff
@@ -797,5 +797,26 @@
     },
     "post_dialog__select_recipients_share_to_everyone": {
         "message": "任何人"
+    },
+    "settings_enable_debug": {
+        "message": "開啓調試模式"
```
Comment (guanbinrui) [inline src/_locales/zh/messages.json]: 開啓圖片編碼模式

File: src/_locales/zh/messages.json [hunk 2]
```diff
@@ -797,5 +797,26 @@
...
+    },
+    "settings_enable_debug_desc": {
+        "message": "Maskbook 會顯示額外的信息以幫助調試"
+    },
+    "settings_image_based_payload": {
+        "message": "將密文編碼進圖片"
```
Comment (guanbinrui) [inline src/_locales/zh/messages.json]: 將密文編碼進圖片而不是編碼進文本

Comment (Jack-Works) [reply src/_locales/zh/messages.json]: The English translation need two lines and it is not easy to make this change

Comment (guanbinrui) [reply src/_locales/zh/messages.json]: OK

**Pull request merged**